Source/ASTableView.mm

```diff
@@ -919,8 +919,13 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 
 - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
-  ASCellNode *node = [_dataController.visibleMap elementForItemAtIndexPath:indexPath].node;
-  CGFloat height = node.calculatedSize.height;
+  CGFloat height = 0.0;
+
+  ASCollectionElement *element = [_dataController.visibleMap elementForItemAtIndexPath:indexPath];
+  ASCellNode *node = element.node;
+  if (element != nil && node != nil) {
+    height = [node layoutThatFits:element.constrainedSize].size.height;
+  }
```

Comment:
Do we not need the else case if they are both available: `node.calculatedSize.height;`

Reply:
No we don't. `-layoutThatFits:` checks and uses the cached size if possible.